notebooks/en/structured_generation.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>"✅ Although the answer is still nonsensical due to the high temperature, the generated output is now in correct JSON <em>format</em>, with the exact keys and types we defined in our grammar!"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>